elasticsearch/src/main/java/org/apache/calcite/adapter/elasticsearch/ElasticsearchRules.java

```diff
@@ -131,6 +131,10 @@ static String stripQuotes(String s) {
         ? s.substring(1, s.length() - 1) : s;
   }
 
+  private static String escapeSpecialSymbols(String s) {
+    return s.replace("\\", "\\\\").replace("\"", "\\\"");
```

Comment:
Question: is there other common case that needs to handled?

Reply:
I didn't find additional cases that should be also handled here.